new_platforms/vscode-extension/assets/baseSolutionTemplateFolder/enc/optee/sub.mk

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+CFLAGS += $(EXTRA_CFLAGS)
+
+CFLAGS +=       \
+	-I$(O)      \
+	-I$(NP_INC) \
+	-I$(OE_INC)
+
+CFLAGS += -DLINUX -DOE_USE_OPTEE
```

Comment:
why is OE_USE_OPTEE needed?   the OE files automatically define it if any of the following are defined:
# if defined(_ARM_) || defined(_M_ARM) || defined(__arm__) || defined(__thumb__) || defined(__aarch64__) || defined(OE_SIMULATE_OPTEE)

Reply:
@HernanGatta, what should I change?

Reply:
@HernanGatta fixed this by removing the OE_USE_OPTEE cmd line option